packages/SequenceTranslator/src/model/data-loading-utils/json-loader.ts

```diff
@@ -0,0 +1,69 @@
+/* Do not change these import lines to match external modules in webpack configuration */
+import * as grok from 'datagrok-api/grok';
+import * as ui from 'datagrok-api/ui';
+import * as DG from 'datagrok-api/dg';
+
+export type KeyToValue = {[key: string]: string};
+
+export type Edges = {
+  [key: string]: KeyToValue
+}
+
+export type FormatDict = {
+  [sourceFormat: string]: {
+    [targetFormat: string]: KeyToValue | Edges
+  }
+}
+
+export type AxolabsStyle = {
+  [index: string]: {
+    fullName: string,
+    symbols: string[],
+    color: string,
+  }
+};
+
+const APP_PATH = 'System:AppData/SequenceTranslator';
+const FORMAT_DICT_FILENAME = 'format-converter-dict.json';
+const AXOLABS_STYLE_FILENAME = 'axolabs-style.json';
+
+export class JsonLoader {
+  private constructor() { }
+
+  private static instance?: JsonLoader;
+  private formatDictionary: FormatDict;
+  private axolabsStyle: AxolabsStyle;
+
+  static getInstance() {
```

Comment:
Why the instance can't simply be initialized and used, what's getInstance for?

Also, does it have to be a class, can't it be a simple function?